Code/Editor/Util/GuidUtil.h

```diff
@@ -62,7 +62,7 @@ inline GUID GuidUtil::FromString(const char* guidString)
     guid.Data1 = 0;
     guid.Data2 = 0;
     guid.Data3 = 0;
-    azsscanf(guidString, "{%8" SCNx32 "-%4hX-%4hX-%2X%2X-%2X%2X%2X%2X%2X%2X}",
+    azsscanf(guidString, "{%8lX-%4hX-%4hX-%2X%2X-%2X%2X%2X%2X%2X%2X}",
```

Comment:
What happened to the 'SCNx32' string?

Reply:
Let me revert this... I needed to change the type in Linux for the GUID to match the one for Windows
In Linux the type GUID is actually declared (in Windows it comes from some windows header). For Windows, the first part is `unsigned long`, whereas in Linux it was declared as `uint32_t` 
That created a problem with SCNx32 in Windows. 
However, seems this change can create a mismatch since Windows is LLP64 and Linux/MacOS is LP64: https://en.cppreference.com/w/cpp/language/types